RunningOrder/Shared/Representables/FocusableNSTextField.swift

```diff
@@ -42,6 +50,8 @@ struct FocusableTextField: NSViewRepresentable {
 
     class Coordinator: NSObject, NSTextFieldDelegate {
         var parent: FocusableTextField
+        var oldFocused: Bool?
```

Comment:
variable usage not clear, you can try to add some comments

Reply:
comments added